uploadNeuroDB/minc_insertion.pl

```diff
@@ -1,4 +1,74 @@
 #! /usr/bin/perl
+
+=pod
+
+=head1 NAME
+
+minc_insertion.pl -- Insert MINC files into the LORIS database system
+
+=head1 SYNOPSIS
+
+perl minc_insertion.pl C<[options]>
+
+Available options are:
+
+-profile     : name of the config file in
```

Comment:
minor indentation issues here and the 2 or 3 inputs below
(see screenshot)
![minc_insertion_indentation](https://user-images.githubusercontent.com/15198379/36229856-8f772906-11a6-11e8-9e30-545138fc2a23.png)

Reply:
@MounaSafiHarab done! Thank you :)